data/workflow/translated_search.sh

```diff
@@ -34,13 +48,27 @@ TARGET="$2"
 TARGET_ORF="$2"
 if [ -n "$TARGET_NUCL" ]; then
 if [ -n "$NO_TARGET_INDEX" ]; then
-    if notExists "${TMP_PATH}/t_orfs_aa.dbtype"; then
-        # shellcheck disable=SC2086
-        "$MMSEQS" extractorfs "$2" "${TMP_PATH}/t_orfs_aa" ${ORF_PAR} \
-            || fail  "extract target orfs step died"
+    if [ "$ORF_SKIP" ]; then
+        if notExists "${TMP_PATH}/t_orfs_aa.dbtype"; then
+            # shellcheck disable=SC2086
+            "$MMSEQS" extractframes "$1" "${TMP_PATH}/t_orfs_nucl" ${EXTRACT_FRAMES_PAR} \
+                || fail  "extractframes died"
+            # we want to avoid this \/
+            # shellcheck disable=SC2086
+            "$MMSEQS" translatenucs "${TMP_PATH}/t_orfs_nucl" "${TMP_PATH}/t_orfs_aa" ${TRANSLATE_PAR} \
+                || fail  "translatenucs died"
+            "$MMSEQS" rmdb "${TMP_PATH}/t_orfs_nucl"
+        fi
+    else
+        if notExists "${TMP_PATH}/t_orfs_aa.dbtype"; then
+            # same here
+            # shellcheck disable=SC2086
+            "$MMSEQS" extractorfs "$2" "${TMP_PATH}/t_orfs_aa" ${ORF_PAR} \
+                || fail  "extract target orfs step died"
+        fi
     fi
-    TARGET="${TMP_PATH}/t_orfs_aa"
-    TARGET_ORF="${TMP_PATH}/t_orfs_aa"
+        TARGET="${TMP_PATH}/t_orfs_aa"
+        TARGET_ORF="${TMP_PATH}/t_orfs_aa"
```

Comment:
Please check if this whitespace should be here. From the diff, this looks wrong.